data/datasets/README.md

```diff
@@ -0,0 +1,16 @@
+# Dialogue files
+
+The YAML files required to train the Rasa DIET classifier can be generated from the annotated dialogues saved in the correct format.
+The generation of these files can be done with this command:
+
+```shell
+  cd usersimcrs/utils
+  python -m annotation_converter_rasa -source PathToAnnotatedDialoguesFile -destination PathToDestinationFolder
+```
+
+It will generate the following files:
```

Comment:
Nit "generate" => "create" just to avoid overuse of generate

Reply:
Done.